engine/src/main/scala/cromwell/engine/workflow/ShadowWorkflowActor.scala

```diff
@@ -101,14 +103,17 @@ class ShadowWorkflowActor(workflowId: WorkflowId, startMode: StartMode, wdlSourc
 
   when(WorkflowUnstartedState) {
     case Event(StartWorkflowCommand, _) =>
-      // TODO: Begin MaterializingWorkflowDescriptor (by outsourcing to MaterializeWorkflowDescriptorActor)
+      val materializeWorkflowDescriptorActor = context.actorOf(MaterializeWorkflowDescriptorActor.props(), name = s"MaterializeWorkflowDescriptorActor-$workflowId")
+      materializeWorkflowDescriptorActor ! MaterializeWorkflowDescriptorActor.MaterializeWorkflow(workflowId, wdlSource)
+      // Kill when done
+      materializeWorkflowDescriptorActor ! PoisonPill
```

Comment:
I haven't used this before, but the documentation says that it treats `PosionPill` just like any other message.. so theoretically the actor should suicide only after sending the result back to us 😕